clients/imodels-client-management/src/base/internal/AxiosRestClient.ts

```diff
@@ -18,6 +18,8 @@ export class AxiosRestClient implements RestClient {
 
   constructor(parseErrorFunc: ParseErrorFunc) {
     this._parseErrorFunc = parseErrorFunc;
+    delete axios.defaults.headers.post["Content-Type"];
```

Comment:
Please add a comment explaining why we are doing this. Also does this solution require to pin `axios` to exact version 1.1.0?

Reply:
There is a problem with transpiling axios on 1.1.3, here's an issue for further information:
https://github.com/axios/axios/issues/5011
it should would work if all axios `import` statements would be changed to `require('axios')` but since it's a bug inside axios i'm not sure if it is worth to do it rather than just to change version and wait for new version